ambition_utils/rrule/models.py

```diff
@@ -214,10 +217,11 @@ def get_rrule_from_params(self, params):
         # Return the rrule
         return rrule(**params)
 
-    def get_next_occurrence(self, last_occurrence=None, force=False):
+    def get_next_occurrence(self, last_occurrence=None, ignore_offset=False, force=False):
```

Comment:
What do you think about renaming `ignore_offset` to be the positive, `include_offset` or `calculate_offset`? It's a personal preference, but I find it easier to read as `if calculate_offset:` rather than `if not ignore_offset`.

Reply:
It's a good suggestion. Updated dfe5e36